sky/serve/core.py

```diff
@@ -124,6 +125,8 @@ def up(
 
     _validate_service_task(task)
 
+    task = policy.Policy().apply_to_task(task)
```

Comment:
A bit unclear if it's in-place since it also returns the task. How about "apply_in_place"?

Reply:
It is not in-place. Either way is fine, but I found not in-place is more commonly seen operations.